app.js

```diff
@@ -15,9 +15,15 @@ var downloadVideo = function(params, callback) {
 		.on('finish', callback);
 
 	// test
-	/*setTimeout(function() {
-		callback();
-	}, Math.ceil(Math.random()*30));*/
+	/*
+	setTimeout(function() {
+	callback();
+	}, Math.ceil(Math.random()*30));
+	*/
+}
+
+var filterFileName = function(fileName){
```

Comment:
I think that we can give better name to this function cause it not only about filter illegal symbols. What about `normalizeFileName` or `fixFileName`?